etc/picongpu/1_radiation.cfg

```diff
@@ -0,0 +1,77 @@
+# Copyright 2013-2021 Rene Widera, Axel Huebl
+#
+# This file is part of PIConGPU.
+#
+# PIConGPU is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# PIConGPU is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with PIConGPU.
+# If not, see <http://www.gnu.org/licenses/>.
+#
+
+##
+## This configuration file is used by PIConGPU's TBG tool to create a
+## batch script for PIConGPU runs. For a detailed description of PIConGPU
+## configuration files including all available variables, see
+##
+##                      docs/TBG_macros.cfg
+##
+
+
+#################################
+## Section: Required Variables ##
+#################################
+
+TBG_wallTime="02:00:00"
+
+TBG_devices_x=1
+TBG_devices_y=1
+TBG_devices_z=1
+
+TBG_gridSize="128 128 128"
+TBG_steps="10"
+
+TBG_periodic="--periodic 1 1 1"
+
+
+#################################
+## Section: Optional Variables ##
+#################################
+
+TBG_radiation="--e_radiation.period 1 --e_radiation.dump 5 \
+               --e_radiation.totalRadiation \
+               --e_radiation.lastRadiation "
```

Comment:
sry that I come again with a change: we should set `e_radiation.numJobs 4` or maybe to `8` to utilize modern GPUs better e.g. A100 and MI100.
The default for this option is `2` but `8` would give us more room for newer GPUs.

Reply:
Additionally, this file is in the wrong directory 🤦